Manifest.toml

```diff
@@ -1,20 +1,29 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.3"
+julia_version = "1.9.0"
```

Comment:
@Bill-Becker  do you think there's any issue with updating the Julia version in the manifest? This change is from https://github.com/NREL/REopt.jl/pull/332

Reply:
We need to update the `[compat]` section of Project.toml if we merge this or there will likely be an error instantiating the package.

Reply:
I don't think we ever want to manually edit the Manifest.toml, as it advises against.

Reply:
@Bill-Becker I don't believe this was a manual update, just the result of local machine-generated changes